### PR TITLE
Fixing npm audit 

### PR DESCRIPTION
**Pull request by wmsbill**
As per as per https://www.npmjs.com/advisories/1076 and https://github.com/storybookjs/marksy/issues/78

### CODE REVIEW

**Comment by lucas-burdell-karmak** [comment]
I also ran into this issue recently and came here to make a PR. Glad you're ahead of me!

It looks like a snapshot file changed. I'm not familiar at all with this repo but that change doesn't look right.

**Comment by wmsbill** [comment]
I thought the same until I saw this commit 2e7f73f

**Comment by lucas-burdell-karmak** [comment]
> I thought the same until I saw this commit [2e7f73f](https://github.com/storybookjs/marksy/commit/2e7f73f59d3ef9ede8a4b1888c1c05fea0d20566)

@ndelangen related to the commit above, is the change we see in the snapshots here not the expected behavior?

**Comment by ndelangen** [comment]
Indeed, this is why I downgraded exactly

File: src/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -203,11 +203,9 @@ exports[`should be able to compile list 1`] = `
 
 exports[`should be able to compile list with html tag 1`] = `
 <div>
-  <font
```
Comment (ndelangen) [inline src/__snapshots__/index.test.js.snap]: This is not correct, unfortunately 😢

**Comment by ndelangen** [comment]
The snapshot should stay the same.

I sunk about 2 hours into this back then, and wasn't able to upgrade without complete breaking the package.

File: src/__snapshots__/index.test.js.snap (hunk 1 above)
Comment (wmsbill) [reply src/__snapshots__/index.test.js.snap]: That is fair. I'll check it out.

**Comment by stof** [comment]
Well, the snapshot for the test `should be able to compile list with html tag` expects a behavior which is not consistent with the Commonmark markdown spec. So upgrades won't ever allow it to stay unchanged (and switching markdown parser will probably not help either, as most parsers are referring to Commonmark now for interoperability).

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "@babel/standalone": "^7.4.5",
     "he": "^1.2.0",
-    "marked": "^0.3.12"
+    "marked": "^0.7.0"
```
Comment (kunyan) [inline package.json]: `marked ` already has `0.8.0` in 3 months ago
https://www.npmjs.com/package/marked

**Comment by stof** [comment]
@ndelangen if the fact that there is some cases changing behavior due to the marked upgrade, maybe this could be released as a new major version of that library, if that's an issue for releasing it as a patch.

File: src/__snapshots__/index.test.js.snap [hunk 3]
```diff
@@ -203,11 +203,9 @@ exports[`should be able to compile list 1`] = `
 
 exports[`should be able to compile list with html tag 1`] = `
```
Comment (ndelangen) [inline src/__snapshots__/index.test.js.snap]: OK, @stof If we remove this testcase, and upgrade, then release as a new major version, I'm ok with that.

**Pull request closed without merging**